packages/core/src/ui/UICanvas.ts

```
@@ -0,0 +1,619 @@
+
+  /**
+   * @internal
+   */
+  @ignoreClone
+  _onEntityModify(flag: EntityModifyFlags): void {
+    switch (flag) {
+      case EntityModifyFlags.SiblingIndex:
+        this._rootCanvas && (this._rootCanvas._hierarchyDirty = true);
+        break;
+      case EntityModifyFlags.UICanvasEnableInScene:
+        if (!this._isRootCanvas) break;
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)